### PR TITLE
Fixed the UI issue and even added new dialogs to different actions

### PR DESCRIPTION
**Pull request by Sami3160**
The constant appearance no-notes.gif by delete note is removed. It was caused by some notes.length field, but it is fixed.
Also to replace that gif i added a simple modal that appears on various actions like add/update/delete.
The modal can automatically disappears after 1500ms and some by 1000ms. or can by explicitly disappeared by clicking on any where on window.
After adding new note, it shows...
Screenshot (76)
after updating any note it shows....
Screenshot (77)
and finally after deleting any note...
Screenshot (78)
Basically i just removed the bug or replaced bug with some modals, i hope you like it😊 also it doesnt change any main functionalities of your project...
please do accept my pull request😄

### CODE REVIEW

**Comment by arpitghura** [comment]
Hey @Sami3160 You have done a great job, but this will not create a good user experience. 

Instead, you can show a toast message in the top right/center.

**Comment by Sami3160** [comment]
> Hey @Sami3160 You have done a great job, but this will not create a good user experience.
> 
> Instead, you can show a toast message in the top right/center.

changes are done in this pr, check the updated preview:
Screenshot (97)
Screenshot (99)
Screenshot (98)

File: main.js [hunk 1]
```diff
@@ -120,4 +155,10 @@ const getData = () => {
 }
 getData();
 
+function toastMessage(msg) {
+    var x = document.getElementById("snackbar");
+    x.className = "show";
+    x.innerHTML=msg;
```
Comment (arpitghura) [inline main.js]: use `innerText` instead of `innerHTML`

File: main.js [hunk 2]
```diff
@@ -91,7 +118,15 @@ const addNewNote = (text = '', title = '') => {
...
     const deleteNote = () => {
         newNote.remove();
+        let newC=currentItemCount();
         updateLSData();
+        if(newC<c){
+            toastMessage("Note deleted successfully!!!")
+
+        }else{
+            toastMessage("Note deleted successfully!!!")
+        }
```
Comment (arpitghura) [inline main.js]: There is no need to check for the count as you are already inside the function call of the delete function.

File: main.js [hunk 3]
```diff
@@ -2,6 +2,23 @@ const addBtn = document.getElementById("addNote");
...
+        ele.title = note.querySelector(".inputHeading").value;
+        if (!(ele.content === "" && ele.title === "" || ele.content === " " && ele.title === " ")) {
+            notes.push(ele);
+        }
+        ele = {};
+    })
+
+    return notes.length
+}
+
```
Comment (arpitghura) [inline main.js]: 1. Return `notes` from the function and reuse this function inside `updateLSData` to fetch all notes.
2. change the function name to something like `getAllNotes`
3. Use the `notes.length` property while fetching the length of the notes or create a separate function to return the length by passing `notes` as the parameter.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Kindly review these reviews.

File: main.js (hunk 1 above)
Comment (Sami3160) [reply main.js]: okay sir, check new pr, all changes are done as you told me

**Review by arpitghura — APPROVED** [review]
Good Work, Sami
